packages/core/src/test/shared/wizards/prompterTester.ts

```diff
@@ -11,8 +11,8 @@ export class PrompterTester {
     private quickPickHandlers: Map<string, (input: TestQuickPick) => void> = new Map()
     private inputBoxHanlder: Map<string, (input: TestInputBox) => void> = new Map()
     private testWindow: TestWindow
-    private calledOrder = 0
-    private report = new Map<string, number>()
+    private callLog = Array<string>()
+    private handlerCallHistory = new Map<string, number>()
```

Comment:
what is the difference between "log" and "history" ?
```suggestion
    private callLogCount = new Map<string, number>()
```

It adds needless friction and entropy to use different names for the same concepts. Please don't do that.

Reply:
Acknowledge.